google/cloud/security/scanner/samples/group_rules.yaml

```diff
@@ -0,0 +1,50 @@
+rules:
+  # Only allow Organization users to be in the Organization groups.
+  - name: sample whitelist for group users
+    mode: whitelist
+    resource:
+      - type: group
+        applies_to: self_and_children
+        resource_ids:
+          - my_customer
```

Comment:
Just a question -- do you build the group "hierarchy" and it all rolls up to "my_customer"?